Lang/Spanish.pj.Lang

```diff
@@ -419,7 +420,7 @@
 //Edit Cheat
 #1027#  "Editar Truco"
 #1028#  "Actualizar Truco"
-#1029#  "El Truco ha sido cambiado.\n\n¿Deseas actualizar?"
+#1029#  "El Truco ha sido cambiado.\n\n¿Quieres actualizar?"

```

Comment:
> Can you name a piece of professional, even commercial software that offers options using "Quieres" instead of "deseas"?

in Windows 10 you can notices that now some words have been translated to "quieres" instead of "deaseas".

and btw I don't use translators, I translate everything myself, just mentioned that I participate in google translation team.